hypervisor/dm/vpci/ivshmem.c

```diff
@@ -28,6 +28,10 @@
 #define	IVSHMEM_IV_POS_REG	0x8U
 #define	IVSHMEM_DOORBELL_REG	0xcU
 
+static struct ivshmem_shm_region mem_regions[8] = {
```

Comment:
Hmmm.  I don't think we've documented there's a limit of 8 regions?  The ivshmem_cfg_h.py script has no limit to the number of regions.

Reply:
Sure, currently this is not reflected in the document, I will update it to ivshmem document later. We currently support up to 8 VMs, so we set 8 regions by default. I think we have to explain this in the relevant documents and do sanity check in the configuration tool.